cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -1,3 +1,4 @@
+/* global _ */
```

Comment:
I would rather not add it to globals for all of platform, because new code should only use RequireJS and `_` would not be undefined/a global in AMD modules.

To get legacy non-requireJS code to pass eslint, a line like this at the top of the file is fine.